mpa/modules/hooks/model_ema_v2_hook.py

```diff
@@ -41,7 +41,10 @@ def __init__(self, ema_decay=0.9995, interval=1, start_epoch=0, dataset_len_thr=
 
     def before_train_epoch(self, runner):
         if not hasattr(self, "use_ema"):
-            self.use_ema = len(runner.data_loader.dataset) > self.dataset_len_thr
+            if isinstance(runner.data_loader.dataset, list):
```

Comment:
are there any chances that runner.data_loader.dataset has list type?

Reply:
Jihwan add dataset class as a list for ComposedDL in this PR